src/ontology/mondo-edit.obo

```diff
@@ -509455,14 +509455,35 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 id: MONDO:0032836
 name: Weiss-Kruszka syndrome
 subset: clingen {source="MONDO:CLINGEN"}
-subset: obsoletion_protected
```

Comment:
@twhetzel I don't think we should remove this "obsoletion-protected" subset annotation.